package.json

```diff
@@ -1,12 +1,15 @@
 {
```

Comment:
Do we also need to update https://github.com/auth0/ad-ldap-connector/blob/master/.nvmrc to reflect node 18?